backend/app/Models/Account.php

```diff
@@ -9,6 +9,14 @@ class Account extends Model
 {
     use HasFactory;
 
+    protected $appends = [ //For adding custom logic
```

Comment:
add comments same with the documentation here, you can also reference that section